src/features/defi/components/Approve/Approve.tsx

```diff
@@ -79,7 +81,17 @@ export const Approve = ({
         </Stack>
         <Stack>
           <Text fontWeight='bold' translation={['modals.approve.header', { asset: asset.name }]} />
-          <Text color='gray.500' translation={['modals.approve.body', { asset: asset.name }]} />
+          <CText color='gray.500'>
+            <Link
+              href={`${asset.explorerAddressLink}${contractAddress}`}
+              color='blue.500'
+              me={1}
+              isExternal
+            >
+              {translate('modals.approve.routerName')}
```

Comment:
I can kind of see why the original `Allow ShapeShift DAO Router to use your %{asset}` was there as a string, but would prefer this to be dynamic now, passed as a prop to the component, and then interpolated.

Given the ramifications of working with the DeFi UI abstraction and the diff size, this doesn't need to be as part of this PR but would be a nice-to-have refactor follow-up